ambuda/utils/proofing_utils.py

```diff
@@ -21,26 +21,29 @@
 """.strip()
 
 
+def transform_blob_to_page(blob: str) -> str:
+    """
```

Comment:
nit: pep 257 suggests putting a one-line summary first -- see https://peps.python.org/pep-0257/ and search "Multi-line docstrings"